src/gui/components/props/input/input.jsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
-const Input = ({ style, className, name, value, onChange, type }) => {
+const Input = ({ style, className, name, value, onChange, type, input }) => {
```

Comment:
`Input` takes an `input`

what about 
 - letting old `type` (for input) and use a new `propType` (or `rawType`) for the propType
 - or `type` and `inputType`

?